.github/workflows/build.yaml

```diff
@@ -7,68 +7,39 @@ on:
     branches:
       - master
   workflow_dispatch:
+    branches:
+      - master
 jobs:
-  build_8:
-    name: Build 8 - ansible 2.9
+  build:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: centos-stream-8
+            shortcut: cs8
+            container-name: el8stream
+          - name: centos-stream-9
+            shortcut: cs9
+            container-name: el9stream
+    name: ${{ matrix.name }}
     container:
-      image: quay.io/centos/centos:stream8
+      image: quay.io/ovirt/buildcontainer:${{ matrix.container-name }}
     steps:
-      - name: Install packages
-        run: yum install -y createrepo_c rpm-build yum-utils git
-      - name: Install ansible packages
-        run: yum install -y ansible-core ansible-test glibc-langpack-en
-      - name: Update all packages
-        run: yum update -y
-
-      - name: Install pip modules
-        run: pip3 install pycodestyle pylint==2.4.4 voluptuous yamllint
-
       - name: Checkout
         uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
 
-      - name: Upgrade pip
-        run: pip3 install --upgrade pip
-
-      - name: Run build.sh
-        run: ./automation/build.sh
-
-      - name: Create DNF repository
-        run: createrepo_c exported-artifacts/
-
-      - name: Upload artifacts
-        uses: actions/upload-artifact@v2
-        with:
-          name: el8_artifacts
-          path: exported-artifacts/
-
-  build_9stream:
-    name: Build 9stream - ansible-core 2.12
-    runs-on: ubuntu-latest
-    container:
-      image: quay.io/centos/centos:stream9
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Install packages
-        run: yum install -y createrepo_c rpm-build yum-utils git ansible-core ansible-test python3-pip
-      - name: Update all packages
-        run: yum update -y
+      - name: Update pip
+        run: pip3 install pip --upgrade
```

Comment:
this should be remove too, buildcontainer has up to date pip from python3-pip

Reply:
When I was running the build manually it was still needed, looks like the python3-pip is not the latest pip.

Reply:
that might be possible, distros usually do not package pipy latest versions...the main question is if you need the latest for whatever reason...

Reply:
We still need it when we try to install the `ansible-lint<5.0.0` it throws `No module named 'setuptools_rust'`

Reply:
yeah, it's quite an old one in there...
# rpm -q python3-pip
python3-pip-9.0.3-22.el8.noarch
# pip3 install pip --upgrade
Successfully installed pip-21.3.1